### PR TITLE
Make the travisci module interactive

### PR DESCRIPTION
**Pull request by anandsudhir**
Make the travisci module interactive
* Add ability to select and open builds
* Add commit message (oneline style) to text displayed in the widget
* Add documentation on the new keyboard shortcuts
* Cleanup some duplicate code

### CODE REVIEW

**Comment by anandsudhir** [comment]
Failed codeclimate again? :(
Could we increase the threshold for the number of duplicate lines a bit? 

I must admit though, I am a Go noob and am trying to learn Go by working with this awesome project.

For instance, the check now fails because the below block appears twice: 
IMHO, this seems a bit too stringent.

```go
func NewWidget(app *tview.Application, pages *tview.Pages) *Widget {
    widget := Widget{
        HelpfulWidget: wtf.NewHelpfulWidget(app, pages, HelpText),
        TextWidget:    wtf.NewTextWidget("TravisCI", "travisci", true),
    }
```

**Comment by senorprogrammer** [comment]
Yep, I agree. The check is too restrictive right now. I'll decrease the sensitivity.

File: wtf/utils.go [hunk 1]
```diff
@@ -113,6 +113,13 @@ func RightAlignFormat(view *tview.TextView) string {
 	return fmt.Sprintf("%%%ds", w-1)
 }
 
+func DefaultFocussedRowColor() string {
```
Comment (senorprogrammer) [inline wtf/utils.go]: This is a very nice improvement.

**Comment by senorprogrammer** [comment]
Latest `master` has a .codeclimate.yml` that relaxes the similar code check. Give that a try.

**Comment by senorprogrammer** [comment]
I've disabled CodeClimate until I have more time to dig into configuring it properly.

**Comment by senorprogrammer** [comment]
Nice work, this is a great addition.

**Pull request merged**